api/handlers/containerhandler.py

```diff
@@ -64,7 +65,7 @@ class ContainerHandler(base.RequestHandler):
             'parent_storage': containerstorage.ProjectStorage(),
             'storage_schema_file': 'session.json',
             'payload_schema_file': 'session.json',
-            'list_projection': {'info': 0, 'analyses': 0},
+            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0, 'subject.lastname': 0},
```

Comment:
Are you sure this is the right layer to exclude personal data? I guess if it's a policy change on all container endpoints, it makes sense, it just surprised me.